packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -38,29 +38,32 @@ abstract class InteractiveInkFeature extends InkFeature {
     @required RenderBox referenceBox,
     Color color,
     VoidCallback onRemoved,
-  }) : assert(controller != null),
-       assert(referenceBox != null),
-       _color = color,
-       super(controller: controller, referenceBox: referenceBox, onRemoved: onRemoved);
+  })  : assert(controller != null),
+        assert(referenceBox != null),
+        _color = color,
+        super(
+            controller: controller,
+            referenceBox: referenceBox,
+            onRemoved: onRemoved);
 
   /// Called when the user input that triggered this feature's appearance was confirmed.
   ///
   /// Typically causes the ink to propagate faster across the material. By default this
   /// method does nothing.
-  void confirm() { }
+  void confirm() {}
 
   /// Called when the user input that triggered this feature's appearance was canceled.
   ///
   /// Typically causes the ink to gradually disappear. By default this method does
   /// nothing.
-  void cancel() { }
+  void cancel() {}
 
   /// The ink's color.
   Color get color => _color;
   Color _color;
+
   set color(Color value) {
-    if (value == _color)
-      return;
+    if (value == _color) return;
```

Comment:
This change is against the style guide (the unchanged one is actually ok) https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#separate-the-if-expression-from-its-statement

There are more cases later.

Reply:
Inkwell have method onHover, i think him must work without onTap.